src/notation/internal/notationinteraction.cpp

```diff
@@ -1711,8 +1711,12 @@ bool NotationInteraction::applyPaletteElement(mu::engraving::EngravingItem* elem
 
             // A text should only be added at the start of the selection
             // There shouldn't be a text at each element
-            for (staff_idx_t staff = firstStaffIndex; staff < lastStaffIndex; staff++) {
-                applyDropPaletteElement(score, firstSegment->firstElement(staff), element, modifiers);
+            if (element->isTempoText()) {
+                applyDropPaletteElement(score, firstSegment->firstElement(firstStaffIndex), element, modifiers);
```

Comment:
I think it should be applied to staff idx 0, at least that's what's done for system text _lines_, so I guess it applies to just system text too. (Maybe it has no functional implications, but at least it's tidier.)